app/controllers/users.js

```diff
@@ -0,0 +1,28 @@
+const { body, validationResult } = require('express-validator');
+
+const { createUser } = require('../services/users');
+const { fieldValidationError } = require('../errors');
+const { creationParamsMapper } = require('../mappers/users');
+const { showSerializer } = require('../serializers/users');
+
+exports.create = [
+  body('user.first_name', 'first_name must be present')
```

Comment:
Why are you checking `user.first_name` instead of `first_name`?

Reply:
Because the expected body is:
```json
{
  "user" :{
      "first_name": "Tincho",
       ...
  }
}
```

Reply:
I don't think the level "user" is necessary. There is no other field at that level.
I would leave the body like:
```
  "first_name":..
```